src/world.rs

```diff
@@ -460,6 +460,42 @@ impl Drop for LazyInsertions {
     }
 }
 
+/// Lazy removals can be used after creating a new
+/// entity in a system. This way, none of the actual
+/// component storages have to be borrowed mutably.
+///
+/// This resource is added to the world by default.
+pub struct LazyRemovals {
```

Comment:
There is no reason to have a second struct, actually. Given that `LazyInsertions` haven't been released yet, let's just update it to `LazyUpdate` and have a method there.